README.md

```diff
@@ -5,39 +5,391 @@
+
+# Expected Output:
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d api-ui ui notificacions pending_publisher
+#   WARN[0000] Found orphan containers ([issuer-vault-1 issuer-postgres-1 issuer-redis-1]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up. 
+#   [+] Running 4/4
+#    ⠿ Container issuer-ui-1                 Started                                                                                                           0.5s
+#    ⠿ Container issuer-api-ui-1             Started                                                                                                           0.5s
+#    ⠿ Container issuer-notificacions-1      Started                                                                                                           0.4s
+#    ⠿ Container issuer-pending_publisher-1  Running  
+```
+
+> For _Apple-M1/M2/Arm_:
+
+```bash
+# FROM: ./
+
+make run-ui-arm;
+# (Equivalent)
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/local/docker-compose.yml up -d api-ui ui notificacions pending_publisher;
+
+# Expected Output:
+#   COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose.yml up -d api-ui ui notificacions pending_publisher
+#   WARN[0000] Found orphan containers ([issuer-vault-1 issuer-postgres-1 issuer-redis-1]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up. 
+#   [+] Running 4/4
+#    ⠿ Container issuer-ui-1                 Started                                                                                                           0.5s
+#    ⠿ Container issuer-api-ui-1             Started                                                                                                           0.5s
+#    ⠿ Container issuer-notificacions-1      Started                                                                                                           0.4s
+#    ⠿ Container issuer-pending_publisher-1  Running  
+```
+
+Everything should be running correctly and we should see the following services:
+
+![Issuer UI API](/docs/3002.png)
 
-### Option 2 - Standalone mode
+#### Testing API UI
+
+To test a request, make sure to set the Authentention section to the following:
+
+**File:** `./env-api`
+
+```bash
+# ...
+
+ISSUER_API_UI_AUTH_USER=user-api
+ISSUER_API_UI_AUTH_PASSWORD=password-api
+```
+
+![Issuer UI API Authentication](/docs/3002-auth.png)
+
+![Issuer UI API Get Credentials](/docs/3002-credentials.png)
+
+#### Testing Admin UI
```

Comment:
This is the literally the issuer node UI. The "admin" only refers to the FE panels of issuer API (:3001) and UI API (:3002).